CONTRIBUTING.rst

```diff
@@ -33,7 +33,7 @@ Pull Request Process
 #. Create a virtual environment and install dependencies including all 
    extras and development dependencies.
    
-   #. Make sure you have ``python>=3.8`` and ``poetry>=1.5.0`` installed
+   #. Make sure you have ``python>=3.9`` and ``poetry>=1.5.0`` installed
```

Comment:
Please also fix this in the `security.md`

Reply:
Thanks, fixed 
b0b170f